solidity/random-beacon/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@keep-network/random-beacon-v2",
+  "version": "0.0.1-dev",
```

Comment:
Don't we want to start versioning with v2?
```suggestion
  "version": "2.0.0-dev",
```
cc @pdyraga

Reply:
115717c